README.md

```diff
@@ -792,6 +824,7 @@ Opera::Operation::Result.new(output: 'success')
 
 ## Opera::Operation::Base - Class Methods
 >
+    - [context|params|dependencies]_[reader|writer|accessor] - predefined methods for easy access
```

Comment:
Should we have dependencies and params only with readers and leave the writer and accessor to context? To enforce params and dependencies as read only?

Reply:
yes! Great idea!